apps/web/utils/constants.ts

```diff
@@ -11,3 +11,11 @@ if (!process.env.NEXT_PUBLIC_BACKEND_URL) {
 
 export const BASE_API_ENDPOINT = process.env.NEXT_PUBLIC_BACKEND_URL;
 export const COURSE_API_ENDPOINT = `${BASE_API_ENDPOINT}/courses`;
+
+export const APPROVED = "APPROVED";
+export const REJECTED = "REJECTED";
+export const PENDING = "PENDING";
+
+export const SUCCESS = "#22bb33";
+export const ERROR = "#bb2124";
+export const INFO = "#aaaaaa";
```

Comment:
Is it possible to use mantine's predefined color ("green", "yellow", "blue") instead if you plan to use this for the badge color?

Reply:
Let's go with default color then. Lemme fix